src/libraries/System.Formats.Tar/src/System/Formats/Tar/TarHeader.Write.cs

```diff
@@ -22,9 +22,11 @@ internal sealed partial class TarHeader
 
         private static ReadOnlySpan<byte> GnuMagicBytes => "ustar "u8;
         private static ReadOnlySpan<byte> GnuVersionBytes => " \0"u8;
+        private static ReadOnlySpan<byte> SeparatorsAsBytes => Encoding.UTF8.GetBytes(PathInternal.DirectorySeparators);
```

Comment:
```suggestion
        private static readonly byte[] s_separatorsAsBytes = Encoding.UTF8.GetBytes(PathInternal.DirectorySeparators);
```
or add a PathInternal.Utf8DirectorySeparators that uses a "..."u8 or the like.  But as this is currently written, every access to SeparatorsAsBytes is going to re-encode and allocate a brand new array.

Reply:
I thought ReadOnlySpan<T> properties were a special case and that they were being cached under the hood.

Reply:
No. If you have `new byte[] { const, const, const, ... }` that's immediately cast to a `ReadOnlySpan<byte>`, that will end up blitting the constant byte data into the assembly's data section and creating a span around it rather than allocating, but that doesn't happen for a method call to `Encoding.UTF8.GetBytes`.